app/controllers/nyre/projects_controller.rb

```diff
@@ -108,9 +129,7 @@ def tracking_method
       "track_nyre_path"
     end
 
-    # copied from subsites, why is it private?
-    private
-
+    # copied from Dcv::Sites::SearchableController
```

Comment:
Why is this copied from Dcv::Sites::SearchableController?

Reply:
it was duped from SubsitesController before; SubsitesController no longer has it (it moved to the mixin). I didn't investigate whether it could be removed here by including the mixin, so I just updated the comment.

Reply:
👍 

At some point maybe we can consider moving it to a mixin shared by both SearchableController and ProjectsController.  That might be a helpful comment to add to avoid potentially revisiting this question again down the line.

Reply:
I'll create a ticket instead of a comment.